BitwardenShared/UI/Vault/Vault/VaultItemSelection/VaultItemSelectionView.swift

```diff
@@ -75,6 +75,13 @@ struct VaultItemSelectionView: View {
 private struct VaultItemSelectionSearchableView: View {
     // MARK: Properties
 
+    /// The message to display when there's no search results.
+    var emptyViewMessage: String {
+        Localizations.thereAreNoItemsInYourVaultThatMatchX(
+            store.state.ciphersMatchingName ?? "--"
+        ) + "\n" + Localizations.searchForAnItemOrAddANewItem
+    }
```

Comment:
🤔 I think we should add a unit or snapshot test for when `store.state.ciphersMatchingName` is `nil`.